### PR TITLE
Community Article - Using Cloud Backup, Cloud Server Images & Cloud F…

### PR DESCRIPTION
**Pull request by BadCrumpany**
…iles for Backups

### CODE REVIEW

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-5716c5b9fb/how-to/).

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-48ebfa0600/how-to/).

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/support.rackspace.com/build-cf37ca22ca/how-to/). :bow:

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-ccd3c8bf2d/how-to/)

**Comment by RackerKate** [comment]
@cyrichardson Please review.

**Comment by rackernexus** [comment]
Can one of the admins verify this patch?

**Comment by cyrichardson** [comment]
@RackerKate On it!

**Comment by cyrichardson** [comment]
@RackerKate Suggestions here: 
[choose-a-rackspace-backup-solution_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/2837519/choose-a-rackspace-backup-solution_cyrEdits.docx)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-91329c6355/how-to/)

File: content/cloud-files/choose-a-rackspace-backup-solution.md [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+or Rackspace Cloud Images is the best fit for backing up your data.
+
+### Backup considerations
+
+This section presents factors that you should consider when you are deciding
+on a backup solution.
+
+#### Functionality
+
+Ensure that you answer the following questions regarding functionality:
```
Comment (catlook) [inline content/cloud-files/choose-a-rackspace-backup-solution.md]: ... following question regarding ...

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c56b909d88/how-to/)

**Pull request merged**